pkg/tests/tasks/security/authentication/auth_test.go

```diff
@@ -16,319 +16,381 @@ package authentication
-			util.Inspect(err, "Failed to get sleep pod name", "", t)
-			cmd := fmt.Sprintf(`curl http://httpbin.%s:8000/ip -s -o /dev/null -w "sleep.%s to httpbin.%s: %%{http_code}"`,
-				to, from, to)
-			msg, err := util.PodExec(from, sleepPod, "sleep", cmd, true)
-			util.Inspect(err, "Failed to get response", "", t)
-			if !strings.Contains(msg, "200") {
-				log.Log.Errorf("Verify setup -- Unexpected response code: %s", msg)
-			} else {
-				log.Log.Infof("Success. Get expected response: %s", msg)
-			}
-		}
-	}
-
-	log.Log.Info("Verify peer authentication policy")
-	util.Shell(`kubectl get peerauthentication --all-namespaces`)
-	log.Log.Info("Verify destination rules")
-	util.Shell(`kubectl get destinationrules.networking.istio.io --all-namespaces -o yaml | grep "host:"`)
-
-	t.Run("Security_authentication_auto_mTLS", func(t *testing.T) {
-		defer util.RecoverPanic(t)
-
-		log.Log.Info("Auto mutual TLS")
-		out, _ := util.Shell(`kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.foo:8000/headers -s | grep X-Forwarded-Client-Cert | sed 's/Hash=[a-z0-9]*;/Hash=<redacted>;/'`)
-		if !strings.Contains(out, "X-Forwarded-Client-Cert") {
-			t.Errorf("Auto mTLS failed to get X-Forwarded-Client-Cert")
-			log.Log.Info("Auto mTLS failed to get X-Forwarded-Client-Cert")
-		}
-
-		out, _ = util.ShellMuteOutputError(`kubectl exec $(kubectl get pod -l app=sleep -n foo -o jsonpath={.items..metadata.name}) -c sleep -n foo -- curl http://httpbin.legacy:8000/headers -s | grep X-Forwarded-Client-Cert`)
-		if strings.Contains(out, "X-Forwarded-Client-Cert") {
-			t.Errorf("Auto mTLS legacy should not get X-Forwarded-Client-Cert")
-			log.Log.Info("Auto mTLS legacy should not to get X-Forwarded-Client-Cert")
-		}
-	})
-
-	t.Run("Security_authentication_enable_global_mTLS_STRICT_mode", func(t *testing.T) {
-		defer util.RecoverPanic(t)
-
-		log.Log.Info("Globally enabling Istio mutual TLS in STRICT mode")
-		util.KubeApplyContents(meshNamespace, util.RunTemplate(PeerAuthPolicyStrictTemplate, smcp))
-		log.Log.Info("Waiting for rules to propagate. Sleep 30 seconds...")
-		time.Sleep(time.Duration(30) * time.Second)
-
-		from := "legacy"
-		ns := []string{"foo", "bar"}
-		for _, to := range ns {
-			sleepPod, err := util.GetPodName(from, "app=sleep")
-			util.Inspect(err, "Failed to get sleep pod name", "", t)
-			cmd := fmt.Sprintf(`curl http://httpbin.%s:8000/ip -s -o /dev/null -w "sleep.%s to httpbin.%s: %%{http_code}"`,
-				to, from, to)
-			msg, _ := util.PodExec(from, sleepPod, "sleep", cmd, true)
-			if strings.Contains(msg, "200") {
-				t.Errorf("Global mTLS expected 000; Got response code: %s", msg)
-				log.Log.Errorf("Global mTLS expected: 000; Got response code: %s", msg)
-			} else {
-				log.Log.Infof("Response 000 as expected: %s", msg)
+	test.NewTest(t).Id("T18").Groups(test.Full, test.InterOp).Run(func(t test.TestHelper) {
```

Comment:
if you reference test you can directly set here "Full, InterOp"

Reply:
Done. Does this make a big difference to you (it doesn't to me, but I'm fine with using a dot import for this).